content/docs/install/upgrade/index.md

```diff
@@ -4,6 +4,65 @@ description: Upgrade your Keptn to 0.17
 weight: 500
 ---
 
+Keptn only supports upgrading from one release to the next;
+you cannot skip releases.
+This page
+
+* Summarizes the upgrade process for the Keptn control plane and Keptn execution plane
+* Includes some notes about special steps required when updating multi-cluster Keptn instances
+* Gives release-specific notes about upgrading from recent releases
+* Includes links to Release Notes and upgrade instructions for older releases,
+  to help customers who must upgrade from older releases
```

Comment:
Maybe add links to the corresponding sections here?

Reply:
Good idea!  Done.